lib/supavisor/application.ex

```diff
@@ -63,7 +68,8 @@ defmodule Supavisor.Application do
         PartitionSupervisor,
         child_spec: DynamicSupervisor, strategy: :one_for_one, name: Supavisor.DynamicSupervisor
       },
-      Supavisor.Vault
+      Supavisor.Vault,
+      {Cachex, name: Supavisor.Cache}
```

Comment:
I believe the vault, partition supervisor, and Cachex should come before the endpoint, because once the endpoint is up, those services should already be running to serve requests.